jxls/src/test/java/org/jxls/common/ContextTest.java

```diff
@@ -16,44 +15,33 @@ public class ContextTest {
 
     @Test
     public void addVars() {
-        Context context = new Context();
-        context.putVar("col", 1);
-        context.putVar("row", 2);
+        Context context = new ContextImpl();
+        context.putVar("col", Integer.valueOf(1));
```

Comment:
@leonate  If you have to edit the sources in a company workspace and such autoboxing things are set as errors there, you have a problem. And otherwise it's more correct that way, I think. But I didn't change all the places. However, it is tedious.

Reply:
It's funny that your IDE complains about this. This topic can probably be debated. It would also be okay for me to revert a commit. I (necessarily) do not edit the sources in the company workspace.